test/metric/metric_value_query.go

```diff
@@ -59,7 +59,8 @@ func (n *MetricValueFetcher) Fetch(namespace, metricName string, metricSpecificD
 		MetricDataQueries: metricDataQueries,
 	}
 
-	log.Printf("Metric data input is : %s", fmt.Sprint(getMetricDataInput))
```

Comment:
Another print with just a pointer (memory) address. Changing it to show something useful.